tests/installation/enable_y2_firstboot.pm

```diff
@@ -20,12 +20,18 @@ use scheduler 'get_test_suite_data';
 
 sub run {
     my $test_data = get_test_suite_data();
+    my $data_path = "yast2/firstboot/";
     select_console 'root-console';
     zypper_call "in yast2-firstboot";
     assert_script_run 'touch /var/lib/YaST2/reconfig_system';
-    # Use default file from package if no custom file was specified
+    # Use default files from package if no custom file was specified
     if ($test_data->{custom_control_file}) {
-        assert_script_run 'wget ' . data_url($test_data->{custom_control_file}) . ' -O /etc/YaST2/firstboot.xml';
+        assert_script_run 'wget ' . data_url($data_path . $test_data->{custom_control_file}) . ' -O /etc/YaST2/firstboot.xml';
+        assert_script_run 'wget ' . data_url($data_path . $test_data->{sysconfig_firstboot}) . ' -O /etc/sysconfig/firstboot';
+        assert_script_run 'mkdir /usr/share/firstboot/custom';
+        foreach ('welcome', 'license', 'finish') {
+            assert_script_run 'wget ' . data_url($data_path . $test_data->{"custom_$_"}) . " -O /usr/share/firstboot/custom/$_.txt";
```

Comment:
we are repeating the .txt here, there is not need for test_data for those three values, for example:
`assert_script_run 'wget ' . data_url($base_path/$file) . " -O /usr/share/firstboot/custom/$file.txt" for qw(welcome license finish);`